views/partials/entries.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="columns">
+    <div class="column">
+        <div class="box">
+            <a href="#"><p class="title">{{title}}</p></a>
+        </div>
+    </div>
+</div>
```

Comment:
This also needs edit and delete icons. I would recommend `<i class="fa-solid fa-pencil"></i>` and `<i
            class="fa-regular fa-trash-can">`